src/briefcase/integrations/docker.py

```diff
@@ -363,32 +412,32 @@ def _dockerize_args(self, args, interactive=False, mounts=None, env=None):
 
         return docker_args
 
-    def run(self, args, env=None, interactive=False, mounts=None, **kwargs):
+    def run(self, args, env=None, cwd=None, interactive=False, mounts=None, **kwargs):
         """Run a process inside a Docker container."""
         # Any exceptions from running the process are *not* caught.
         # This ensures that "docker.run()" behaves as closely to
         # "subprocess.run()" as possible.
-        self.tools.logger.info("Entering Docker context...", prefix=self.app.app_name)
```

Comment:
Do you think it also makes sense to wrap the Docker image build process with this same context?

```python
        with self.tools.input.wait_bar("Building Docker image..."):
            with self.tools.logger.context("Docker"):
                try:
                    self.tools.subprocess.run(
                        [
                            "docker",
                            "build",
```